packages/vscode-extension/package.nls.json

```diff
@@ -157,6 +157,30 @@
     "teamstoolkit.commandsTreeViewProvider.manageCollaboratorTitle": "Manage Collaborator",
     "teamstoolkit.commandsTreeViewProvider.manageCollaboratorDescription": "Manage M365 Teams App (with Microsoft Entra app) Collaborators",
     "teamstoolkit.commandsTreeViewProvider.addWebpartTitle": "Add SPFx Web Part",
+    "teamstoolkit.commandsTreeViewProvider.officeDevDeployTitle": "Deploy",
+    "teamstoolkit.commandsTreeViewProvider.officeDevDeployDescription": "Deploy",
```

Comment:
Now we have Deploy but no Provision? How is the cloud resources being created? (Or what does deploy do here?)

Reply:
We do not provide provision and deployment functionality. The "Deploy" command just refers to a link that opens the official doc about how to deploy an office add-in.

Reply:
> Now we have Deploy but no Provision? How is the cloud resources being created? (Or what does deploy do here?)

@MuyangAmigo The Deploy is only to open the deployment related url.